pages/BatchPublications/FileFormat.md

```diff
@@ -10,45 +10,40 @@ Batch files are stored and transferred in Apache Parquet format.
 
 ## Specification Status
 
-| Version | Status |
----------- | ---------
-| 0.1     | Draft |
+| Version | Status   |
+| ------  | -------- |
+| 1.0     | Proposed |
```

Comment:
is 1.0 proposed or is it some other more permanent status? Also just a question - how are we determining status. Do we have that documented somehere?

Reply:
We don't have any official numbering yet, but I think it is reasonable to consider 1.0 for Betanet. We should discuss this with the team however.

Status names are here: https://spec.dsnp.org/ under "Specification status definitions"